app/main.py

```diff
@@ -14,6 +14,5 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    phrase = phrase.lower()
-    count = phrase.count(letter.lower())
+    count = phrase.lower().count(letter.lower())
     return count
```

Comment:
It's an unnecessary variable `count`, as Yurii said, make it in one line with `return`